src/gatsby/src/styles/search.scss

```diff
@@ -310,6 +310,10 @@ div[class$="-singleValue"] {
 }
 
 .park-heading-text {
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;  
```

Comment:
:O I don't know anything about these CSS rules but I'm reading that they're supported in Edge/Firefox/Chrome 👍 
If this project is set up to auto-prefix styles, maybe there's a way to write it without the `-webkit` prefixes?

(Looks like maybe there isn't, since the rules are still a working draft...?)

Reply:
Yup, we don't have it ... but it'd be a good idea to have a rule at some point.